.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Bug Report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<!-- 
+    NOTE: Did you check the FAQs (https://one-dark.gitbook.io/jetbrains/help/faqs)
+    to see if your problem already has a work around?
+-->
+
+**Describe the bug**
+A clear and concise description of what the bug is.
```

Comment:
Typically, I've seen these use comments so that users submitting bug reports don't have to delete a bunch of text.
```suggestion
<!-- A clear and concise description of what the bug is. -->
```